lib/p2p/packets/types/DisconnectingPacket.ts

```diff
@@ -15,6 +18,46 @@ class DisconnectingPacket extends Packet<DisconnectingPacketBody> {
   public get direction() {
     return PacketDirection.Unilateral;
   }
+
+  public static deserialize(binary: Uint8Array): DisconnectingPacket | undefined {
```

Comment:
The `deserialize` and `validate` methods aren't using the arrow methods we use elsewhere. It doesn't really matter but it would be nice to keep them consistent.